crates/autopilot/src/database/order_events.rs

```diff
@@ -16,21 +17,65 @@ impl super::Postgres {
     /// elaborate error handling is not necessary because this is just
     /// debugging information.
     pub async fn store_order_events(&self, events: &[(OrderUid, OrderEventLabel)]) {
```

Comment:
Is this method still used? For what use cases would we store all events?

Reply:
Yes, it's used where orders are filtered in the code itself instead. `store_market_order_events` is used for cases where we operate with order uids only without having access to the order class.